src/condition.cpp

```diff
@@ -1484,6 +1492,38 @@ std::function<std::string( const dialogue & )> conditional_t::get_get_string( co
         return [option]( const dialogue & d ) {
             return get_option<std::string>( option.evaluate( d ) );
         };
+    } else if( jo.get_string( "mutator" ) == "valid_technique" ) {
+        std::vector<str_or_var> blacklist;
+        if( jo.has_array( "blacklist" ) ) {
+            for( const JsonValue &jv : jo.get_array( "blacklist" ) ) {
+                blacklist.push_back( get_str_or_var( jv, "blacklist" ) );
+            }
+        }
+
+        bool crit = jo.get_bool( "crit", false );
+        bool dodge_counter = jo.get_bool( "dodge_counter", false );
+        bool block_counter = jo.get_bool( "block_counter", false );
+
+        return [blacklist, crit, dodge_counter, block_counter]( const dialogue & d ) {
+            std::vector<matec_id> bl;
+            for( const str_or_var &sv : blacklist ) {
+                bl.push_back( matec_id( sv.evaluate( d ) ) );
```

Comment:
```suggestion
            std::vector<matec_id> bl;
            bl.reserve( blacklist.size() );
            for( const str_or_var &sv : blacklist ) {
                bl.emplace_back( matec_id( sv.evaluate( d ) ) );
```

clang-tidy is really not happy lately :P

Reply:
Sorry I thought it was green when I looked!